tests/presenters/data_generators.py

```diff
@@ -132,3 +133,96 @@ def get_offset(self) -> Optional[int]:
 
     def get_limit(self) -> Optional[int]:
         return self.limit
+
+
+class PlanSummaryGenerator:
+    def create_plan_summary(
+        self,
+        plan_id: Optional[UUID] = None,
+        is_active: Optional[bool] = None,
+        planner_id: Optional[UUID] = None,
+        planner_name: Optional[str] = None,
+        product_name: Optional[str] = None,
+        description: Optional[str] = None,
+        active_days: Optional[int] = None,
+        timeframe: Optional[int] = None,
+        production_unit: Optional[str] = None,
+        amount: Optional[int] = None,
+        means_cost: Optional[Decimal] = None,
+        resources_cost: Optional[Decimal] = None,
+        labour_cost: Optional[Decimal] = None,
+        is_public_service: Optional[bool] = None,
+        price_per_unit: Optional[Decimal] = None,
+        is_available: Optional[bool] = None,
+        is_cooperating: Optional[bool] = None,
+        cooperation: Union[None, UUID, Literal["None"]] = "None",
```

Comment:
Why not `cooperation: Optional[UUID] = None`? What do we use the extra "None" for?